okhttp/src/test/java/okhttp3/CacheCorruptionTest.kt

```diff
@@ -52,25 +62,29 @@ class CacheCorruptionTest(
     HostnameVerifier { name: String?, session: SSLSession? -> true }
   private val cookieManager = CookieManager()
 
-  @BeforeEach fun setUp() {
+  @BeforeEach
+  fun setUp() {
     platform.assumeNotOpenJSSE()
     platform.assumeNotBouncyCastle()
     server.protocolNegotiationEnabled = false
-    cache = buildCache(File("/cache/"), Int.MAX_VALUE.toLong(), fileSystem)
+    val loggingFileSystem = LoggingFilesystem(fileSystem)
```

Comment:
NEAT

Reply:
I hope we can come up with ways to make it simple to debug problems with the Cache.  And eventually make it safe to use the Cache, as in fail early if you are doing something broken, or with a clear error at runtime.  I like the FileSystem API for this reason.

All towards less bug reports and stackoverflow questions with bizarre errors.